src/TestReporter/Entity/CiInfo.php

```diff
@@ -25,7 +25,7 @@ public function toArray()
             return $this->tddiumProperties();
         }
 
-        if (isset($_SERVER["CI_NAME"]) && preg_match('/codeship/i', $_SERVER["CI_NAME"])) {
+        if (isset($_SERVER["CI_NAME"]) && false !== stripos($_SERVER["CI_NAME"], 'codeship')) {
```

Comment:
This does change functionality w.r.t. case-insensitivity. We should try to confirm somehow that `CI_NAME` would never be (e.g.) `CodeShip`.